src/styles/blocks/calendar.scss

```
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  justify-content: center;
```

```suggestion
  justify-content: left;
```
Calendar should be left-aligned